### PR TITLE
Updated supported versions in CSI Operator docs

### PR DESCRIPTION
**Pull request by panigs7**
# Description
Operator docs changes to update versions reference.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/243|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Comment by shanmydell** [comment]
@panigs7 : Could you please rebase the branch, looks like it has been branched from main

File: content/docs/csidriver/installation/operator/_index.md [hunk 1]
```diff
@@ -50,21 +50,21 @@ If you have installed an old version of the `dell-csi-operator` which was availa
 #### Full list of CSI Drivers and versions supported by the Dell CSI Operator
 | CSI Driver         | Version   | ConfigVersion  | Kubernetes Version   | OpenShift Version     |
 | ------------------ | --------- | -------------- | -------------------- | --------------------- |
-| CSI PowerMax       | 2.0.0     | v2.0.0         | 1.20, 1.21, 1.22     | 4.6 EUS, 4.7, 4.8     |
 | CSI PowerMax       | 2.1.0     | v2.1.0         | 1.20, 1.21, 1.22     | 4.8, 4.8 EUS, 4.9     |
 | CSI PowerMax       | 2.2.0     | v2.2.0         | 1.21, 1.22, 1.23     | 4.8, 4.8 EUS, 4.9     |
-| CSI PowerFlex      | 2.0.0     | v2.0.0         | 1.20, 1.21, 1.22     | 4.6 EUS, 4.7, 4.8     |
+| CSI PowerMax       | 2.3.0     | v2.3.0         | 1.21, 1.22, 1.23     | 4.8, 4.8 EUS, 4.9     |
```
Comment (rensyct) [inline content/docs/csidriver/installation/operator/_index.md:55]: Openshift supported versions for driver 2.3.0 are 4.9 and 4.10. Please do the same for all drivers

Comment (panigs7) [reply content/docs/csidriver/installation/operator/_index.md:55]: @rensyct addressed the review comment in PR https://github.com/dell/csm-docs/pull/201.

**Comment by panigs7** [comment]
Please refer PR https://github.com/dell/csm-docs/pull/201

**Pull request closed without merging**